src/js/BookReader.js

```diff
@@ -940,58 +959,37 @@ BookReader.prototype.drawLeafsThumbnail = function(seekIndex) {
 };
 
 BookReader.prototype.lazyLoadThumbnails = function() {
-  // We check the complete property since load may not be fired if loading from the cache
-  this.$('.BRlazyloading').filter('[complete=true]').removeClass('BRlazyloading');
-
-  var loading = this.$('.BRlazyloading').length;
-  var toLoad = this.thumbMaxLoading - loading;
+  const self = this;
 
-  var self = this;
+  const batchImageRequestByRow = ($images) => {
+    utils.wait().then(() => {
```

Comment:
Can you add a comment about why we're waiting here?

```suggestion
    await utils.wait(500);
```